README.md

```diff
@@ -29,7 +29,7 @@ If you know the specific node name that you're interested in, you can exec into
 
 ```bash
 NODE_NAME="my-node-name"
-POD_NAME=$(kubectl -n kube-system get pods --field-selector spec.nodeName=${NODE_NAME} -ojsonpath='{.items[0].metadata.name}')
+POD_NAME=$(kubectl -n kube-system get pods --field-selector spec.nodeName=${NODE_NAME} -ojsonpath='{.items[?(@.metadata.labels.name=="doks-debug")].metadata.name}')
```

Comment:
I think a better approach is to use the label that should uniquely identify doks-debug.

Personally, I generally prefer using custom columns over dealing with JSON whenever I can, so this is how I'd go about it:

```shell
POD_NAME=$(kubectl get pod -l name=doks-debug --field-selector spec.nodeName=${NODE_NAME} --no-headers -o custom-columns=:.metadata.name)
```

Reply:
@dhij could you check in on that? Take care to add `-n kube-system` still if you use Timo's command.